scripts/natural_language_inference/decomposable_atten.py

```diff
@@ -0,0 +1,100 @@
+# pylint: disable=C0103,E1101,R0914
```

Comment:
Remove disable and fix the problems if possible. If not, please state the reasons